test/Microsoft.TemplateEngine.Authoring.CLI.IntegrationTests/VerifyCommandArgsTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this seems like a unit tests and should be moved to unit tests.
it would be good to add parsing unit tests as well (as in `TemplateEngine.CLI`)